jmxtrans/pom.xml

```diff
@@ -354,8 +350,7 @@
 									fi</script>
 							</preremoveScriptlet>
 							<requires>
-								<require>java</require>
-								<require>/usr/bin/lsb_release</require>
+								<require>java >= 1.7</require>
```

Comment:
It looks like the `JCommanderArgumentParser` library has a java 1.7 dependency though.  I saw the following error when I ran with jdk6:

```
Exception in thread "main" java.lang.NoClassDefFoundError: java/nio/file/Path
	at com.beust.jcommander.internal.DefaultConverterFactory.<clinit>(DefaultConverterFactory.java:66)
	at com.beust.jcommander.JCommander.<clinit>(JCommander.java:167)
	at com.googlecode.jmxtrans.cli.JCommanderArgumentParser.parseOptions(JCommanderArgumentParser.java:35)
	at com.googlecode.jmxtrans.JmxTransformer.main(JmxTransformer.java:118)
Caused by: java.lang.ClassNotFoundException: java.nio.file.Path
	at java.net.URLClassLoader$1.run(URLClassLoader.java:217)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:205)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:323)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:294)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:268)
	... 4 more
Exception in thread "main" java.lang.NoClassDefFoundError: java/nio/file/Path
	at com.beust.jcommander.internal.DefaultConverterFactory.<clinit>(DefaultConverterFactory.java:66)
	at com.beust.jcommander.JCommander.<clinit>(JCommander.java:167)
	at com.googlecode.jmxtrans.cli.JCommanderArgumentParser.parseOptions(JCommanderArgumentParser.java:35)
	at com.googlecode.jmxtrans.JmxTransformer.main(JmxTransformer.java:118)
Caused by: java.lang.ClassNotFoundException: java.nio.file.Path
	at java.net.URLClassLoader$1.run(URLClassLoader.java:217)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:205)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:323)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:294)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:268)
	... 4 more
```